jobs/ostree-compose.yml

```diff
@@ -59,40 +59,30 @@
               which repo of the package being updated
     builders:
         - shell: |
-            cat << EOF > ${WORKSPACE}/job.properties_running
-                topic=org.centos.prod.ci.pipeline.compose.running
-                build_url=$BUILD_URL
-                compose_url=http://artifacts.ci.centos.org/artifacts/fedora-atomic/${fed_branch}/ostree
-                compose_rev=
-                build_id=$BUILD_ID
-                ref=fedora/${fed_branch}/x86_64/atomic-host
-                rev=${rev}
-                namespace=rpms
-                repo=$fed_repo
-                status=success
-                test_guidance=
-            EOF
+            if [ "$fed_branch" = "master" ]; then
+                echo "fed_branch=rawhide" > ${WORKSPACE}/job.properties
+            fi
         - inject:
-            properties-file: ${WORKSPACE}/job.properties_running
+            properties-file: ${WORKSPACE}/job.properties
         - jms-messaging:
             override-topic: org.centos.prod.ci.pipeline.compose
```

Comment:
I do it in rpmbuild.yml because I use jms-messaging twice, and if I ever have to change anything I only want to have to change it in one place.  But you are 100% correct, it is entirely unnecessary in rpmbuild_trigger.yml and I will change it.